src/components/Notification/index.tsx

```diff
@@ -0,0 +1,153 @@
+import * as React from 'react';
+import { Box, styled, useThemeProps } from '@mui/material';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faClose } from '@fortawesome/free-solid-svg-icons/faClose';
+import { faArrowUpRightFromSquare } from '@fortawesome/free-solid-svg-icons/faArrowUpRightFromSquare';
+import { shape } from '@balena/design-tokens';
+import { MUILinkWithTracking } from '../MUILinkWithTracking';
+import { IconButtonWithTracking } from '../IconButtonWithTracking';
+import { useEffect } from 'react';
+import { useAnalyticsContext } from '../../contexts/AnalyticsContext';
+
+export interface NotificationProps {
+	campaignId: string;
+	children: React.ReactNode;
+	illustration?: React.ReactNode;
+	title: string;
+	linkHref: string;
+	linkLabel: string;
+	onClose?: (event: React.SyntheticEvent<any>) => void;
+}
+const NotificationRoot = styled(Box, {
+	name: 'Notification',
+	slot: 'root',
+})<{ ownerState: NotificationProps }>(({ theme, ownerState }) => ({
+	alignItems: 'center',
+	backgroundColor: 'white',
+	borderRadius: shape.border_radius.md.value + 'px',
+	boxShadow: '0px 0px 8px 0px rgba(35, 68, 94, 0.3)', // TODO add a design token for box shadows
+	display: 'flex',
+	gap: theme.spacing(3),
+	maxWidth: 400,
+	padding: theme.spacing(3),
+	position: 'relative',
```

Comment:
Is this only needed for the small breakpoint of `NotificationContent`? If so we should only apply this during that breakpoint as well, right?

Reply:
That's a good question, my hunch is that it's better to keep it there at all time than to add more media queries to 'toggle' it.

Reply:
removed because of the changes in the close icon (no need for absolute positioning anymore)